nomt/src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub use nomt_core::proof;
 pub use nomt_core::trie::{KeyPath, LeafData, Node};
 pub use options::Options;
 
-mod beatree;
+pub mod beatree;
```

Comment:
Ideally, we don't expose this to users. If there is absolutely no way, a comment should explain why there is this internal part sticking out.

Reply:
Yes, I will add a comment for sure. Also, thinking about exposing this module to the user is not great at all. So, I will find a way to put at least one conditional compilation here to expose it only during benchmarks